src/deluge/model/output.h

```diff
@@ -146,6 +148,11 @@ class Output {
 	void endArrangementPlayback(Song* song, int32_t actualEndPos, uint32_t timeRemainder);
 	bool recordingInArrangement;
 
+	virtual ModelStackWithAutoParam*
+	getModelStackWithParam(ModelStackWithTimelineCounter* modelStack, Clip* clip,
+	                       int32_t paramID = deluge::modulation::params::kNoParamID,
+	                       deluge::modulation::params::Kind paramKind = deluge::modulation::params::Kind::NONE) = 0;
```

Comment:
```suggestion
	                       deluge::modulation::params::ParamType paramID,
	                       deluge::modulation::params::Kind paramKind) = 0;
```

I don't think it ever makes sense to call this without providing a `paramID` and `paramKind` so there's no need for default arguments.